spec/features/end_to_end_spec.rb

```diff
@@ -49,6 +49,11 @@
       expect(page).to have_content('Add New Work')
     end
 
+    it 'can see the license wizard on the new work form' do
+      visit new_curation_concerns_work_path
+      expect(page).to have_content('Scholar@UC License Descriptions')
```

Comment:
This text is unrelated to the license wizard and as-is the test should pass regardless of whether the button is showing. Need to fix this to look for the text from the button label, e.g. 'License Wizard'.